fe2-android/app/src/main/java/com/github/dedis/popstellar/ui/detail/LaoDetailViewModel.java

```diff
@@ -109,7 +109,8 @@ public class LaoDetailViewModel extends AndroidViewModel
       new MutableLiveData<>();
   private final MutableLiveData<SingleEvent<Boolean>> mOpenNewRollCallEvent =
       new MutableLiveData<>();
-  private final MutableLiveData<SingleEvent<HomeViewModel.HomeViewAction>> mOpenRollCallEvent = new MutableLiveData<>();
+  private final MutableLiveData<SingleEvent<HomeViewModel.HomeViewAction>> mOpenRollCallEvent =
```

Comment:
In general, try to split in separate commits reformatting without code changes from actual code changes 😉 it makes it easier to review

Reply:
I'll try to be more careful. It was not on purpose, it's (a misuse of) auto-formatting